<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeScreen</class>
 <widget class="QWidget" name="HomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #333333; color: white;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="statusBar">
     <property name="maximumHeight">
      <number>50</number>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #222222;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QProgressBar" name="batteryProgressBar">
        <property name="maximumWidth">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="format">
         <string>%p%</string>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar {
   border: 1px solid #555;
   border-radius: 5px;
   text-align: center;
}
QProgressBar::chunk {
   background-color: #00aaff;
   width: 10px;
   margin: 0.5px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="clockLabel">
        <property name="text">
         <string>6:32 PM | November 14</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="insulinProgressBar">
        <property name="maximumWidth">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="format">
         <string>%v u</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar {
   border: 1px solid #555;
   border-radius: 5px;
   text-align: center;
}
QProgressBar::chunk {
   background-color: #00aaff;
   width: 10px;
   margin: 0.5px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="cgmGraphFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #222222;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="glucoseValueLabel">
        <property name="text">
         <string>88 mg/dL</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #00aaff;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="trendValueLabel">
        <property name="text">
         <string>â†’</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #00aaff;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="graphFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">/* Graph will be drawn programmatically */</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeFrameLabel">
        <property name="text">
         <string>3 HRS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="iobFrame">
     <property name="maximumHeight">
      <number>50</number>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #222222;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="iobLabel">
        <property name="text">
         <string>IOB:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="iobValueLabel">
        <property name="text">
         <string>1.1 units</string>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #00aaff;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="iobTimeLabel">
        <property name="text">
         <string>1:09 hrs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QPushButton" name="tButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>120</height>
      </size>
     </property>
     <property name="text">
      <string>t</string>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
   background-color: #00aaff;
   color: white;
   border-radius: 10px;
}
QPushButton:pressed {
   background-color: #0088cc;
}</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="optionsButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>OPTIONS</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/gear.png</normaloff>:/icons/gear.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
   background-color: #444444;
   color: white;
   border-radius: 5px;
}
QPushButton:pressed {
   background-color: #333333;
}</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="bolusButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>BOLUS</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/droplet.png</normaloff>:/icons/droplet.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
   background-color: #00aaff;
   color: white;
   border-radius: 5px;
}
QPushButton:pressed {
   background-color: #0088cc;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 